modules/phase_field/tests/DeformedGrain/DeformedGrain.i

```diff
@@ -0,0 +1,120 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 32
+  ny = 0
+  nz = 0
+  xmin = 0
+  xmax = 64
+[]
+
+[GlobalParams]
+  block = 0
+  op_num = 2
+  ndef = 1
```

Comment:
for being used as a global parameter this name is definitely _not_ specific/clear enough